src/lib/y2storage/blk_device.rb

```diff
@@ -58,6 +58,10 @@ class BlkDevice < Device
     #   @raise [Storage::Exception] if the devicegraph provided as first
     #     argument is not the probed one
     #
+    #   @raise [DeviceNotFoundByName] if no device is found
```

Comment:
I don't think this is true. The wrapped version of the method (the one that is described here) will never raise those two kinds of exceptions. It will return `nil`, as it was already documented.